projects/mac_ai/testing/prepare_mac_ai.py

```
@@ -173,6 +178,16 @@ def prepare_binary():
     if config.project.get_config("prepare.podman.machine.enabled"):
         podman_machine.configure_and_start(base_work_dir, force_restart=False)
 
+    if config.project.get_config("prepare.remoting.publish"):
+        if not config.project.get_config("exec_list.pre_cleanup_ci"):
+           raise ValueError("Cannot publish the remoting libraries if not preparing from a clean environment")
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)